### PR TITLE
KiCad.KiCad.Lite version 5.1.12_1

### PR DESCRIPTION
**Pull request by swoga**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34927)

### CODE REVIEW

File: manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KiCad.KiCad.Lite
+PackageVersion: 5.1.12_1
+PackageLocale: en-US
+Publisher: KiCad
+PublisherSupportUrl: https://www.kicad.org/help/
+Author: KiCad
+PackageName: KiCad
```
Comment (ItzLevvie) [inline manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml:10]:
```suggestion
PackageName: KiCad Lite
```
To make sure it doesn't conflict with the non-Lite version.

File: manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Publisher: KiCad
+PublisherSupportUrl: https://www.kicad.org/help/
+Author: KiCad
+PackageName: KiCad
+PackageUrl: https://www.kicad.org/
+License: GPL-3.0-or-later
+LicenseUrl: https://gitlab.com/kicad/code/kicad/-/blob/master/LICENSE
+Copyright: Copyright (c) 2014-2021 KiCad Developers
+ShortDescription: KiCad is a free software suite for electronic design automation. It facilitates the design of schematics for electronic circuits and their conversion to PCB designs.
+Moniker: kicad
```
Comment (ItzLevvie) [inline manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml:16]:
```suggestion
Moniker: kicad-lite
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml (hunk 1 above)
Comment (swoga) [reply manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml:10]: Thanks for the correction was a copy&paste error

File: manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml (hunk 2 above)
Comment (swoga) [reply manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml:16]: I have copied this from the existing Lite package, but I agree, a different value should be assigned here.

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**